dts/arm/microchip/mec172xnsz.dtsi

```diff
@@ -718,8 +718,7 @@
 			reg = <0x40070000 0x400>;
 			interrupts = <91 2>;
 			girqs = < MCHP_XEC_ECIA(18, 1, 10, 91) >;
-			pcrs = <4 8>;
-			clocks = <&pcr 1 0>;
+			clocks = <&pcr 4 0>;
```

Comment:
@scottwcpg isn't main clock 96MHz in mec172x or is this for 4 an enum and not just divisor?

Reply:
The value 4 is the bit position in PCR registers for sleep enable, etc.  Not related to QMSPI clock divider.